lib/eval.coffee

```diff
@@ -34,6 +34,31 @@ module.exports =
           @showError r, result.highlights
         notifications.show "Evaluation Finished"
 
+  # get documentation or methods for the current word
+  toggleMeta: (type) ->
+    editor = atom.workspace.getActiveTextEditor()
+    [word, range] = @getWord editor
+    # if we only find numbers or nothing, return prematurely
+    if word.length == 0 || !isNaN(word) then return
+    client.msg type, {code: word}, ({result}) =>
+      view = if result.type then result.view else result
+      view = @ink.tree.fromJson(view)[0]
+      @ink.links.linkify view
+      r = @ink?.results.toggleUnderline editor, range,
+        content: view
+        clas: 'julia'
+
+  # gets the word and its range in the `editor` which the last cursor is on
+  getWord: (editor) ->
+    cursor = editor.getLastCursor()
+    # The following line is kinda iffy: The regex may or may not be well chosen
+    # and it duplicates the efforts from atom-language-julia. It might be
+    # better to select the current word via finding the smallest <span> containing
+    # the cursor which also has `function` or `macro` as its class.
+    range = cursor.getCurrentWordBufferRange({wordRegex: /[\w_!´]*\.?@?[\w_!´]+/})
```

Comment:
Hmm. I think it would be great to fix the grammar so it acts more like the parser, but that might be harder.

Given that you have a goal for what the scope selector should ideally do, do you think you could make the changes to the grammar?